cmd/kube-apiserver/app/server.go

```diff
@@ -157,11 +155,12 @@ func updateEtcdOverrides(overrides []string, storageVersions map[string]string,
 		}
 
 		servers := strings.Split(tokens[1], ";")
+		etcdConfig.ServerList = servers
```

Comment:
if we're reusing the struct inside the overrides loop, shouldn't we work with a copy inside each loop, rather than stomping the server list each time through?